### PR TITLE
Pop-up-Window

### PR DESCRIPTION
**Pull request by den-cloud**
In this PR, the following changes were made:

Added the modal container in the index.html file
Added styling for the modal container in the stylesheet.css file
Updated each project in a modal from the array in the portfolio.js file
Ensured there was no linters error
This feature of the project gives also some interactivity to my portfolio website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,7 +209,7 @@ <h2 class="project-title hide-for-mobile">Uber Navigation</h2>
                         <li class="tag">css</li>
                         <li class="tag">javaScript</li>
                     </ul>
-                    <button class="button" type="button">See Project</button>
+                    <button class="button see-project" type="button">See Project</button>
```
Comment (SajjadAhmad14) [inline index.html:212]: Buttons should be in the center of the project details card. Moreover please add a bit more space between icon in the button and text. Kindly see the below screenshot for reference.
buttons

File: index.html [hunk 2]
```diff
@@ -101,7 +101,7 @@ <h2 class="project-title">Tonic</h2>
                         <li class="tag">css</li>
```
Comment (SajjadAhmad14) [inline index.html:101]: Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and make the write professional commit messages.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Denzel Nkomo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is followed.
- [ ] README Is professional.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (den-cloud) [inline index.html:212]: Please have look at figma template 1 and see the how its style on the page

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Denzel, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 :shipit: .
Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]**:_

- [ ] OPTIONAL
Kindly endeavor to include the `.vscode` directory in the `.gitignore` so as to keep your repository cleaner

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**